.github/workflows/deploy-images-dockerhub.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy Images to DockerHub
+
+on:
```

Comment:
Should we not perhaps run this when a tag is created in git, then it doesn't have to run manually.

Reply:
Hi @rcrichton . Yes, I did think of this, but my worry was that it would trigger for any random tag/release creation (which might not be what we want). Throught it might be better to be more specific in regards to this. p.s you can also configure a local docker repo, of which when configure images are pushed to this on merge automatically (with the short commit hash as the tag). It's only when pushing to docker hub when someone will need to be more specific.